src/main/java/com/microsoft/sqlserver/jdbc/IOBuffer.java

```diff
@@ -1597,9 +1598,16 @@ private void validateServerNameInCertificate(X509Certificate cert) throws Certif
      *        Server Host Name for SSL Handshake
      * @param port
      *        Server Port for SSL Handshake
+     * @param clientCertificate
+     *        Client certificate path
```

Comment:
Is this absolute path? Can applications also provide relative path?

Reply:
It accepts both relative and absolute path.